### DISCUSSION TITLE
关于 formatjs swc 插件的使用

### DISCUSSION THREAD

**Discussion by iwtem** [web-infra-dev/rsbuild#2586] [Q&A]
最近在给项目做国际化，引入了 formatjs，其中有个 `@formatjs/swc-plugin` 插件，官网的使用步骤如下：
![image](https://github.com/web-infra-dev/rsbuild/assets/76606053/886b2ebc-743b-4437-96b4-0715ff8d7a12)

使用过程中会报出如下的错误
![image](https://github.com/web-infra-dev/rsbuild/assets/76606053/b3f8dda6-32db-4ec2-9643-ac25115382c2)

请问是否是使用的不正确，还是不支持该插件

**Comment by chenjiahan** [comment]
可以检查下是不是 SWC 版本与 SWC 插件版本不一致的问题：

![image](https://github.com/web-infra-dev/rsbuild/assets/7237365/85ab66ef-3486-4059-b66c-826de99f591e)

https://rsbuild.dev/zh/config/tools/swc#%E6%B3%A8%E5%86%8C-swc-%E6%8F%92%E4%BB%B6
